advanced/security.md

```diff
@@ -2,46 +2,141 @@
+
+## Reporting TinyMCE security issues
+
+{{site.companyname}} values the work of security researchers in improving the security of technology products worldwide. We welcome researchers who wish to responsibly disclose vulnerabilities in our products or systems. Note that we do not offer any “bug bounty” program or any form of payment for disclosed vulnerabilities. If you would like to report a vulnerability, please email `infosec@tiny.cloud`.
+
+## What we do to maintain TinyMCEs' security
+
+To protect {{site.productname}}s' users, {{site.companyname}}:
+
+- Patches Cross-Site Scripting (XSS) vulnerabilities,
+- Keeps {{site.productname}}s' dependencies up to date, and
+- Provides information how to configure a Content Security Policy that works with {{site.productname}}.
+
+### Scripts and XSS vulnerabilities
+
+{{site.productname}} filters out most Cross-Site Scripting (XSS) content such as scripts from the editor content. For additional security, {{site.companyname}} recommends passing the editor content through server-side filters such as [HTMLPurifier](http://htmlpurifier.org/).
+
+### Keeping dependencies up-to-date
+
+To protect our users, {{site.companyname}} ensures that TinyMCEs' dependencies are updated before the next version (major, minor, or patch) is released.
+
+### Configuring Content Security Policy (CSP) for TinyMCE
 
 {{site.productname}} can be used with a [CSP](https://content-security-policy.com/) header. When using a CSP, the following directives are **required** for {{site.productname}} to function:
 
-| Directives | Requirements |
-|------------|--------------|
-| script-src 'self' *.tinymce.com *.tiny.cloud;          | Scripts are sometimes loaded as script element with an src attribute.
-| connect-src 'self' *.tinymce.com *.tiny.cloud blob:;         | XMLHttpRequest are required by some services such as spellchecking and PowerPaste.
-| img-src 'self' *.tinymce.com *.tiny.cloud data: blob:; | Images within the editor are sometimes base64 encoded, blob URLs, or proxied through the {{site.cloudname}} service.
-| style-src 'self' 'unsafe-inline' *.tinymce.com *.tiny.cloud;        | Styles are used for inline formatting (such as underline, font colors, etc.) and the positioning of user interface elements.
-| font-src 'self' *.tinymce.com *.tiny.cloud;            | Fonts are used for icons in the UI and is loaded from external files.
+| Directives                                                   | Requirements                                                                                                                 |
+| ------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------- |
+| `script-src 'self' _.tinymce.com _.tiny.cloud;`                | Scripts are sometimes loaded as script element with an src attribute.                                                        |
```

Comment:
Why was the asterisks changed to underscores? as that's definitely not correct, they should be asterisks.

Reply:
my bad, a vscode extension formatted the markdown. Will fix

Reply:
fixed